### PR TITLE
feat(dedicated): add technical details tile to dashboard

### PR DESCRIPTION
**Pull request by frenautvh**
| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | internal MANAGER-5537
| License          | BSD 3-Clause

## Description

add technical details tile to bare metal cloud dashboard

draft : missing some api data

### CODE REVIEW

**Review by radireddy — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/technical-details/technical-details.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<oui-tile
+    class="h-100"
+    heading="{{:: 'dedicated_server_dashboard_technical_details_heading' | translate }}"
+>
+    <oui-tile-definition
+        term="{{:: 'dedicated_server_dashboard_technical_details_cpu' | translate }}"
+    >
+        <oui-tile-description> </oui-tile-description>
```
[the remark this answers is not in the document]
Comment (frenautvh) [reply packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/technical-details/technical-details.html:8]: template is not done yet, i'll complete it when i'll have all the needed data from api (it's still a draft)

File: packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/technical-details/technical-details.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        </oui-tile-description>
+    </oui-tile-definition>
+    <oui-tile-definition
+        term="{{:: 'dedicated_server_dashboard_technical_details_ram' | translate }}"
+    >
+        <oui-tile-description>
+            <span data-ng-bind=":: $ctrl.formattedRAM"></span>
+        </oui-tile-description>
+    </oui-tile-definition>
+    <!--
```
Comment (cbourgois) [inline packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/technical-details/technical-details.html:19]: Should we keep-it ?

File: packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.html [hunk 3]
```diff
@@ -475,171 +482,14 @@
...
-                        </div>
-                    </div>
-                </div>
-            </div>
+        <!-- SUBSCRIPTIONS -->
+        <div class="col-md-4" data-ng-if="!$ctrl.technicalDetails">
+            <dedicated-server-dashboard-subscriptions
+                change-owner-url="$ctrl.changeOwnerUrl"
+                dedicated-server="$ctrl.dedicatedServer"
+                world-part="$ctrl.worldPart"
```
Comment (cbourgois) [inline packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.html:490]: Or use `Environment` in the component ?

**Review by cbourgois — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/technical-details/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  "dedicated_server_dashboard_technical_details_gpu": "GPU",
+  "dedicated_server_dashboard_technical_details_system_disk_one": "Disque système",
+  "dedicated_server_dashboard_technical_details_data_disk_one": "Disque de données",
+  "dedicated_server_dashboard_technical_details_cache_disk_one": "Disque de cache",
+  "dedicated_server_dashboard_technical_details_extension_card_one": "Carte d'extension",
+  "dedicated_server_dashboard_technical_details_system_disk_many": "Disques système",
+  "dedicated_server_dashboard_technical_details_data_disk_many": "Disques de données",
+  "dedicated_server_dashboard_technical_details_cache_disk_many": "Disques de cache",
+  "dedicated_server_dashboard_technical_details_extension_card_many": "Cartes d'extension",
+  "dedicated_server_dashboard_technical_details_cpu_frequency_unit": "Ghz",
```
Comment (antleblanc) [inline packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/technical-details/translations/Messages_fr_FR.json:15]:
```suggestion
  "dedicated_server_dashboard_technical_details_cpu_frequency_unit": "GHz",
```

File: packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/technical-details/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  "dedicated_server_dashboard_technical_details_ram_size_unit": "Go",
+  "dedicated_server_dashboard_technical_details_ram_frequency_unit": "Mhz",
```
Comment (antleblanc) [inline packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/technical-details/translations/Messages_fr_FR.json:17]:
```suggestion
  "dedicated_server_dashboard_technical_details_ram_frequency_unit": "MHz",
```

**Review by antleblanc — CHANGES_REQUESTED** [review]

**Comment by Heiiji** [comment]
i have merge my modification approved on https://github.com/ovh/manager/pull/4188

**Comment by frenautvh** [comment]
closed in favor of : https://github.com/ovh/manager/pull/4313

**Pull request closed without merging**